<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CorrectionWindow</class>
 <widget class="QMainWindow" name="CorrectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CorrectionWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="beforeView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="afterView">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="beforeLabel">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>50</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Before Normalizing</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="afterLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>After Normalizing</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="importButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Image (.czi)</string>
    </property>
   </widget>
   <widget class="QSlider" name="layerSlider">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>26</width>
      <height>391</height>
     </rect>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="layerLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LayerSlider</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="thresholdMode">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply Threshold in Normalizing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="alignButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Align</string>
    </property>
   </widget>
   <widget class="QComboBox" name="channelSelectMenu">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>104</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>All</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>#1: Red</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>#2: Green</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>#3: Blue</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>949</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
